gm4_ambrosia/pack.mcmeta

```diff
@@ -0,0 +1,28 @@
+{
+  "pack":{
+    "pack_format":0,
+    "description":"A Gamemode 4 Module"
+  },
+  "module_name":"Bees",
+  "module_id":"ambrosia",
+  "site_description":"To bee or not to bee - time to add some invertebrate friends to your world!",
+  "site_categories":["Bees"],
```

Comment:
What other module falls under this category?

Reply:
There's another module in the works that will be released under this category!